specs/experimental/fault-proof/bond-incentives.md

```diff
@@ -0,0 +1,31 @@
+# Bond Incentives
+
+Bonds is an add-on to the core [Fault Dispute Game](./fault-dispute-game.md). The core game mechanics are
+designed to ensure honesty as the best response to winning subgames. By introducing financial incentives,
+Bonds makes it worthwhile for honest challengers to participate.
+
+- Moves
+- Subgame Resolution
+
+## Moves
+
+Moves must be adequately bonded to be added to the FDG. This document does not specify a
+scheme for determining the minimum bond requirement. FDG implementations should define a function
+computing the minimum bond requirement with the following:
+
+```solidity
+function getRequiredBond(Position _movePosition) public pure returns (uint256 requiredBond_)
+```
+
+As such, attacking or defending requires a check for the `getRequiredBond()` amount against the bond
+attached to the move. To incentivize participation, the minimum bond should cover the cost of a possible
+counter to the move being added. Thus, the minimum bond depends only on the position of the move that's added.
+
+## Subgame Resolution
```

Comment:
I think this is missing discussion on the freeloader vs opposing teams as mentioned in `specs/experimental/fault-proof/honest-challenger-fdg.md`

Reply:
Added a section on the incentives implied by the leftmost payoff rule. It covers what claims are defensible and why an honest challenger will want to ensure their claim is the unique leftmost claim.